src/Sulu/Bundle/PageBundle/Trash/PageTrashItemHandler.php

```diff
@@ -172,12 +199,16 @@ public function restore(TrashItemInterface $trashItem, array $restoreFormData =
                 $localizedPageAccessor->set('uuid', $uuid);
             }
 
+            $lastModified = $localeData['lastModified'] ? new \DateTime($localeData['lastModified']) : null;
+
             $localizedPage->setTitle($localeData['title']);
             $localizedPage->setResourceSegment($localeData['resourceSegment']);
             $localizedPage->setSuluOrder($localeData['suluOrder']);
             $localizedPage->setLocale($locale);
             $localizedPage->setCreator($localeData['creator']);
             $localizedPage->setCreated(new \DateTime($localeData['created']));
+            $localizedPage->setLastModifiedEnabled($localeData['lastModifiedEnabled']);
```

Comment:
This should not be set, as we should not save it. Just have a getter for the serialzer and to give it to the frontend:

```suggestion
```